Recycled Bitmap RuntimeException fix.

DominikAngerer opened the pull request:
Will fix - Fatal Exception: java.lang.RuntimeException
Canvas: trying to use a recycled bitmap android.graphics.Bitmap@34cf39

Commit:
Recycled Bitmap RuntimeException fix.

Will fix - Fatal Exception: java.lang.RuntimeException
Canvas: trying to use a recycled bitmap android.graphics.Bitmap@34cf39

mikepenz commented on app/src/main/java/com/mikepenz/unsplash/views/adapters/ImageAdapter.java:
```diff
@@ -76,6 +76,7 @@ public ImagesViewHolder onCreateViewHolder(ViewGroup viewGroup, int position) {
     public void onBindViewHolder(final ImagesViewHolder imagesViewHolder, final int position) {
 
         final Image currentImage = mImages.get(position);
+        imagesViewHolder.image.setImageBitmap(null);
```
there's no "image" in the holder
-> should be: "imagesViewHolder.imageView.setImageBitmap(null);"
also please add the source and the note about this one as comment above the line. 

Also please to the .setImageBitmap under the setDrawingCache line

Commit:
Update ImageAdapter.java

Switched position of setImageBitmap & changed to your variable name (sorry for that)

Commit:
Update ImageAdapter.java
